error: ordered comparison between pointer and zero ('FILE *' (aka '__sFILE *') and 'int')
--- src/3rdParty/salomesmesh/src/DriverDAT/DriverDAT_R_SMDS_Mesh.cpp.orig	2017-07-17 12:27:47.000000000 -0500
+++ src/3rdParty/salomesmesh/src/DriverDAT/DriverDAT_R_SMDS_Mesh.cpp	2018-02-28 23:52:54.000000000 -0600
@@ -52,7 +52,7 @@
    ****************************************************************************/
   char *file2Read = (char *)myFile.c_str();
   FILE* aFileId = fopen(file2Read, "r");
-  if (aFileId < 0) {
+  if (aFileId == NULL) {
     fprintf(stderr, ">> ERREUR : ouverture du fichier %s \n", file2Read);
     return DRS_FAIL;
   }
--- src/3rdParty/salomesmesh/src/DriverDAT/DriverDAT_W_SMDS_Mesh.cpp.orig	2017-07-17 12:27:47.000000000 -0500
+++ src/3rdParty/salomesmesh/src/DriverDAT/DriverDAT_W_SMDS_Mesh.cpp	2018-02-28 23:55:04.000000000 -0600
@@ -38,7 +38,7 @@
   
   char *file2Read = (char *)myFile.c_str();
   FILE* aFileId = fopen(file2Read, "w+");
-  if (aFileId < 0) {
+  if (aFileId == NULL) {
     fprintf(stderr, ">> ERREUR : ouverture du fichier %s \n", file2Read);
     return DRS_FAIL;
   }
